entity-framework/core/managing-schemas/migrations/index.md

```diff
@@ -1,294 +1,188 @@
 ---
-title: Migrations - EF Core
+title: Migrations Overview - EF Core
 author: bricelam
 ms.author: bricelam
-ms.date: 10/05/2018
+ms.date: 05/06/2020
 uid: core/managing-schemas/migrations/index
 ---
-# Migrations
+# Migrations Overview
 
-A data model changes during development and gets out of sync with the database. You can drop the database and let EF create a new one that matches the model, but this procedure results in the loss of data. The migrations feature in EF Core provides a way to incrementally update the database schema to keep it in sync with the application's data model while preserving existing data in the database.
+In real life projects, data models change as features get implemented: new entities or properties are added and removed, and databases schema needs to be changed accordingly to be kept in sync with the application. The migrations feature in EF Core provides a way to incrementally update the database schema to keep it in sync with the application's data model while preserving existing data in the database.
 
-Migrations includes command-line tools and APIs that help with the following tasks:
+At a high level, migrations functions in the following way:
 
-* [Create a migration](#create-a-migration). Generate code that can update the database to sync it with a set of model changes.
-* [Update the database](#update-the-database). Apply pending migrations to update the database schema.
-* [Customize migration code](#customize-migration-code). Sometimes the generated code needs to be modified or supplemented.
-* [Remove a migration](#remove-a-migration). Delete the generated code.
-* [Revert a migration](#revert-a-migration). Undo the database changes.
-* [Generate SQL scripts](#generate-sql-scripts). You might need a script to update a production database or to troubleshoot migration code.
-* [Apply migrations at runtime](#apply-migrations-at-runtime). When design-time updates and running scripts aren't the best options, call the `Migrate()` method.
+* When a data model change is introduced, the developer uses EF Core tools to add a corresponding migration describing the updates necessary to keep the database schema in sync. EF Core compares the current model against a snapshot of the old model to find out the differences, and generates migration source files; the files are kept in your project's source control like any other source file.
```

Comment:
"find out the differences" -> "determine the differences"
"the files are kept" -> "the files can be tracked"